iris/static/js/lecturer_session.js

```diff
@@ -1,38 +1,42 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
```

Comment:
I don't recall if we agreed on semicolons or not. Standard StandardJS doesn't use it, but we can do whatever we want I guess. Will let this pass anyways because we're inconsistent on this in general, will fix later.